conference.js

```diff
@@ -1657,6 +1710,14 @@ export default {
                     micDeviceId: null
                 })
                 .then(([stream]) => {
+                    if (this.isAudioOnly()) {
+                        return stream.mute()
+                            .then(() => stream);
+                    }
+
+                    return stream;
+                })
+                .then(stream => {
```

Comment:
I talked with hristo and we're okay with not changing this right now. I couldn't come up with a way to avoid another .then without nesting .thens even further due to the mute function being  async.